src/Themes/VuetifyTheme/Migrations.cs

```diff
@@ -40,6 +42,7 @@ public int Create()
             CompatibilityBanner();
             VFooter();
             UpdateToMultiTextField();
+            UpdateFrom1();
```

Comment:
No need to add `UpdateFrom1()`... Orchard knows to apply UpdateFrom1 if it needs to.

Also, visually, it would be best if the `UpdateFrom1` method block was moved underneath `Create`.